sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+### Withdrawals (Redeeming sBTC)
+A clarity call initiates the withdrawal process that successfully completed after 6 sortitions paying to the specified Bitcoin address. Failure cases can be resolved earlier than 6 sortitions, but some will only fail after the sBTC Bootstrap Signer attempts to create the withdrawal transaction.
+
+**Six Sortitions for Withdrawal:** The sBTC Bootstrap signers will only trigger the withdrawal on Bitcoin after the Stacks withdrawal has been confirmed by 6 sortitions. Bitcoin transactions cannot be constructed such that they’re valid for only a specific Bitcoin fork, nor can they be forcefully taken back after they’re created, and a Bitcoin fork can cause a Stacks reorg such that the initial withdrawal request becomes invalid. The sBTC Bootstrap Signers will need to wait for the withdrawals to be sufficiently final in order to make the withdrawal on Bitcoin without risk of allowing a double withdrawal from the Stacks Blockchain. The withdrawal procedure will need to track the number of valid sortitions after the withdrawal to verify that the Stacks transaction is final, and 6 is the standard choice for Bitcoin finality.
+
+#### Withdrawal Flow
+The main steps of the sBTC withdrawal flow are as follows:
+1. **Withdrawal request:** An sBTC holder calls the withdraw-request function in the .sbtc contract.
+    - This transfers the requested amount of sBTC to the .sbtc contract & mints the user a non-transferable locked-sBTC as a placeholder.
+2. **Withdrawal accept:** If accepted, the following happens:
+    - The signers create a "Withdrawal accept" transaction on Bitcoin which returns the requested amount to the designated address.
+    - Once the Bitcoin transaction is confirmed, the signers must then call into the sBTC protocol to mark it as complete - this is done through a Stacks transaction.
+    - If successful, the resulting Stacks transaction will record the withdrawal request as complete & will accordingly burn the user’s locked-sBTC.
+3. **Withdrawal reject:** If instead the request is rejected, the sBTC signers will call the withdraw-reject function in the .sbtc smart contract. This function does the following:
+    - Returns the sBTC to the holder.
+    - Records the signer votes.
+
+To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+
+## Auxiliary Features
+Auxiliary features of the sBTC Bootstrap protocol are described below.
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this issue, sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
```

Comment:
Is this in-scope?  Asking because this doesn't sound unique to sBTC -- it's possible today for someone to sponsor another's transaction in return for some non-STX asset.

Reply:
It's part of the overall sBTC bootstrap phase "project" that's relevant to the end users. It'll follow the proposal that Marvin made a while ago and we'll provide them to application developers to make their lives easier. It's not something that's traditionally in a SIP but in describing the whole of what sBTC Bootstrap is to a developer and user it's nice to have that described here.

Reply:
I still don't think this is in-scope.  This is an _application_ of sBTC, but it's not part of sBTC proper.  You could put this in an "example applications" section of something, but someone reading this SIP in order to learn how to implement a compliant sBTC system doesn't need to do this, right?